src/uiSelectController.js

```diff
@@ -257,8 +254,6 @@ uis.controller('uiSelectCtrl',
       if (ctrl.dropdownPosition === 'auto' || ctrl.dropdownPosition === 'up'){
         $scope.calculateDropdownPos();
       }
-
-      $scope.$broadcast('uis:refresh');
```

Comment:
Can't users listen on it? 🤔 I don't use events much in angular so not completely sure!